### DISCUSSION TITLE
Adding Darwinia Chain to Guild.xyz

### DISCUSSION THREAD

**Discussion by 2xJDubs** [darwinia-network/darwinia#1332] [Ideas]
I think Darwinia should talk to Guild.xyz about supporting Darwinia Smart Chain so we can use Guild for tokengating of roles based on NFTs on DSC.  Darwinia already has a DAO guild on the site so this makes a lot of sense and will allow allow the DCDAO to grant a DAO role on Discord using Guild.xyz.

https://guild.xyz/darwinia

**Comment by SasoLithops** [comment]
I support this. Would be great for any further Discord activities since lots starting to happen there.

**Comment by hackfisher** [comment]
Moved to https://github.com/ringecosystem/integration/issues/20 , join the queue for integration team cc @fewensa @hujw77
